Dokumentationen/requirements/system-wide_requirements.adoc

```diff
@@ -80,6 +80,11 @@ Stakeholder haben Anforderungen. Ziele der Stakeholder sind Teil der Anforderung
 
 *** Zur Überprüfung wird eine Testperson die Anwendung mit einem Abstand von 50cm Entfernung zum Bildschirm nutzen. 
 
+** *NFAU-1.6*: Freitextfelder sind in ihrer Zeichenkapazität begrenzt. 
+*** Der Name des Dummy-Experiments fasst maximal 75 Zeichen.
+*** Der Name des Studiengangs fasst maximal 75 Zeichen. 
+*** Der Buchungskommentar fasst 200 Zeichen
```

Comment:
Wir sollten dafür einen Testcase schreiben. Wollen wir das in diesem PR mit machen oder später (dann vielleicht ein Issue dazu anlegen?)?

Reply:
@Greyhound993 was sagst du?

Reply:
Ja finde ich eine tolle Idee, ich leg den Issue gleich an :)